lib/rucio/daemons/replicarecoverer/suspicious_replica_recoverer.py

```diff
@@ -358,46 +358,33 @@ def run_once(heartbeat_handler: Any, younger_than: int, nattempts: int, vos: Opt
 
                         file_metadata_datatype = str(file_metadata["datatype"])
                         file_metadata_scope = str(file_metadata["scope"])
-                        datatype_found_in_file = False
-                        scope_found_in_file = False
-                        wildcard_found_in_file = False
                         action = ""
                         for i in json_data:
```

Comment:
You should use a variable name that are more self-explanatory, e.g. policy instead of i